Cake.Recipe/Content/packages.cake

```diff
@@ -33,20 +33,13 @@ BuildParameters.Tasks.DotNetCorePackTask = Task("DotNetCore-Pack")
         - GetFiles(BuildParameters.SourceDirectoryPath + "/**/*.Tests.csproj")
         - GetFiles(BuildParameters.SourceDirectoryPath + "/packages/**/*.csproj");
 
-    var msBuildSettings = new DotNetCoreMSBuildSettings()
-                            .WithProperty("Version", buildVersion.SemVersion)
-                            .WithProperty("AssemblyVersion", buildVersion.Version)
-                            .WithProperty("FileVersion",  buildVersion.Version)
-                            .WithProperty("AssemblyInformationalVersion", buildVersion.InformationalVersion);
-
-    if (BuildParameters.BuildAgentOperatingSystem != PlatformFamily.Windows)
+    var msBuildSettings = new DotNetCoreMSBuildSettings();
```

Comment:
@AdmiringWorm should the same comment as before be added here:

```
// We need to clone the settings class, so we don't
// add additional properties to every other task.
```

Reply:
Will do on next pr update